template/src/main.js

```diff
@@ -1,7 +1,9 @@
+import './set-public-path' //eslint-disable-line will complain about order of imports
```

Comment:
Semicolon:

```suggestion
import './set-public-path'; //eslint-disable-line will complain about order of imports
```

Reply:
Also, could you explain which eslint rule fails because of the order of imports? And whether this is enabled by default with vue-cli or if it's one you've opted into for your projects